compiler/rustc_session/src/options.rs

```diff
@@ -150,6 +151,9 @@ top_level_options!(
 
         target_triple: TargetTriple [TRACKED],
 
+        /// Effective logical environment used by `env!`/`option_env!` macros
+        logical_env: FxHashMap<String, String> [UNTRACKED],
```

Comment:
Any change in `logical_env` can potentially change in compiled code so I think this should be tracked.
```suggestion
        logical_env: FxHashMap<String, String> [TRACKED],
```

Reply:
:+1: